bin/license.py

```diff
@@ -0,0 +1,376 @@
+    "oro:oro:2.0.8": "(Apache License, Version 1.1) ORO (oro:oro:2.0.8 - https://mvnrepository.com/artifact/oro/oro)",
+    "org.hyperic:sigar:1.6.5.132": "(Apache License, Version 2.0) Sigar (org.hyperic:sigar:1.6.5.132 - https://github.com/hyperic/sigar)"
+}
+licenses_describe_map = {
+    "Apache 2.0 License":
+        """The following components are provided under the Apache License. See project link for details.
+The text of each license is the standard Apache 2.0 license.
+""",
+    "MIT License": """The following components are provided under the MIT License. See project link for details.
+The text of each license is also included at licenses/LICENSE-[project].txt.
+""",
+    "BSD License": """The following components are provided under a BSD license. See project link for details.
+The text of each license is also included at licenses/LICENSE-[project].txt.
+""",
+    "CC0-1.0 License": """The following components are provided under the CC0-1.0 License. See project link for details.
+The text of each license is also included at licenses/LICENSE-[project].txt.
+""",
+    "CDDL License": """The following components are provided under the CDDL License. See project link for details.
+The text of each license is also included at licenses/LICENSE-[project].txt.
+""",
+    "Eclipse Public License": """The following components are provided under the Eclipse Public License. See project link for details.
+The text of each license is also included at licenses/LICENSE-[project].txt.
+""",
+    "Public Domain License": """The following components are provided under the Public Domain License. See project link for details.
+The text of each license is also included at licenses/LICENSE-[project].txt.
+""",
+    "Mozilla Public License Version 2.0": """The following components are provided under the Mozilla Public License. See project link for details.
+The text of each license is also included at licenses/LICENSE-[project].txt.
+""",
+    "Unicode License": """The following components are provided under the Unicode License. See project link for details.
+The text of each license is also included at licenses/LICENSE-[project].txt.
+""",
+    "Go License": """The following components are provided under the Go License. See project link for details.
+The text of each license is also included at licenses/LICENSE-[project].txt.
+""",
+    "Other License": """The following components are provided under some minority Licenses. See project link for details.
+The text of each license is also included at licenses/LICENSE-[project].txt.
+"""
+}
+licenses_map = {
+    "Apache 2.0 License": [],
+    "MIT License": [],
+    "BSD License": [],
+    "CC0-1.0 License": [],
+    "CDDL License": [],
+    "Eclipse Public License": [],
+    "Public Domain License": [],
+    "Mozilla Public License Version 2.0": [],
+    "Unicode License": [],
+    "Go License": [],
+    "Other License": []
+}
+
+for _ in licenses:
```

Comment:
Due to we use this variable in the next, I think we better named it for easier to maintain
```suggestion
for license in licenses:
```

Reply:
Emm.. I believe the `_` can simplify the code, which is easier to maintain, right

Reply:
I think `_` is usually used for unused variables, why do you think using `_` is easier maintain?

Reply:
> I think `_` is usually used for unused variables, why do you think using `_` is easier maintain?

Yeah, the best practices is for unused variables or for list/tuple comprehension